DbTables/inc/DbTable.hh

```diff
@@ -38,6 +38,8 @@ namespace mu2e {
     // approx size in bytes - overridden by derived class
     virtual std::size_t size() const =0;
     std::size_t baseSize() const { return _csv.capacity(); }
+    // if table needs to be sorted
+    virtual const std::string orderBy() const {return std::string();}
```

Comment:
Should we make this pure virtual to force derived classes to supply an orderBy function? Or are there cases in which there is no meaningful order so an empty string makes sense as a default?

Reply:
Most tables do not need to be ordered, so I did not want to force each table to declare an order.  But this a bit of an open question until all tables have been reviewed and I can think about it a bit more.  I will consider this question as part of the process.

Reply:
Thanks.   I will approve and merge now.